bindings/python/test/test_arrow.py

```diff
@@ -237,19 +242,18 @@ def test_pymongo_error(self):
 
     def test_write_schema_validation(self):
         schema = {
-            "data": int64(),
-            "float": float64(),
-            "datetime": timestamp("ms"),
-            "string": string(),
-            "bool": bool_(),
+            k.__name__: v(True)
+            for k, v in _TYPE_NORMALIZER_FACTORY.items()
+            if k.__name__ not in ("ObjectId, Decimal128")
         }
         data = Table.from_pydict(
             {
-                "data": [i for i in range(2)],
+                "Int64": [i for i in range(2)],
```

Comment:
Should this test include nulls for each field type too?

Reply:
https://jira.mongodb.org/browse/ARROW-94